index.html

```diff
@@ -8,19 +8,18 @@
   <title>Awesome Books Website</title>
 </head>
 <body>
-  <main>
+  <main class="flex direction arrange">
   <section class="top">
     <h1>Awesome Books</h1>
   </section>
-  <section class="bookings">
-    <h2>
-      Books
-    </h2>
+  <section class="bookings flex direction">
+    <h2>All Awesome Books</h2>
   </section>
-  <section>
-    <h2>Add a Book</h2>
+  <section class="ending flex direction arrange">
+    <span class="line"></span>
+    <h2>Add a New Book</h2>
     <form id="formAdd" method="post">
-      <ul>
+      <ul class="flex direction arrange">
```

Comment:
Please follow the wireframe. The list of books should have a 100% width. Form inputs should have a bit more height and width and the form button should be positioned on the right of the form. Please decrease the margin between the `Awesome Books` title and the books list.

Yours:
![width](https://user-images.githubusercontent.com/35504149/143407071-36efba9a-1ed9-4a9e-95d4-102fdad3c041.png)

Wireframe:
![o-width](https://user-images.githubusercontent.com/35504149/143407234-3eddb569-6d92-4176-95f9-ccb943c0dd48.png)
![form-width](https://user-images.githubusercontent.com/35504149/143407267-ebdebc5c-3403-4912-93e0-5426482ea161.png)

Reply:
The review is well received but I like to point out the requirements 

- You can use the styles of your choice (colors, fonts, backgrounds, etc.), but try to follow the general layout.
- Concentrate on the general layout, rather than the details.
I still increased the width to 50% as in my opinion 100% width will make the booklist look a bit off. 
I increased the size of the form and repositioned the button

Reply:
This issue has been addressed well. Great job! :pray: